### ISSUE TITLE
New activity fails to display records

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#873]
I've been asked to set up an activity for Salisbury Plain Training Area, in connection with a forthcming article in the MoD Sanctuary magazine. I've created an indicative boundary in the "My Sites" area of the iRecord Support user account, and then created an Activity using the boundary from My Sites.

Activity summary:
https://www.brc.ac.uk/irecord/activities/summary?group_id=1773&implicit=

The site I've created has location ID 245153.

The summary is not displaying any records, and when I look at the list of sites in My SItes none of them are showing any records at all, so there seems to be a general problem with sites for the iRecord Support account. (On my own account My Sites seems to be displaying correctly.)

This is somewhat urgent as the article deadline is at the beginning of July and it would be useful to be able to refer to this activity.

**Comment by johnvanbreda** [comment]
The Salisbury Plain Training Area location seems to be in the middle of the sea - https://www.brc.ac.uk/irecord/my-site-details?location_id=245153. Any idea how this happened? I just tried to create a site under the iRecord Support account called Tentsmuir and it seemed to draw OK. Note that these sites show zero records simply because the My Sites page shows your own records, not all records, and the iRecord support account doesn't have any.

**Comment by kitenetter** [comment]
Hmm, when I first went to My Sites yesterday I got one of the SQL timeout messages, but I carried on working on the SPTA boundary and it seemed to save okay. And after saving it I went back in and edited it a couple of times and at that point it was displaying correctly on the map.

And when I set up the Activity the boundary was also displaying correctly against the background map, although when I look at it today it has shifted into the sea as you say.

So I guess I need to start all over again. Annoying that it let me get so far before becoming apparent that there was a problem. Closing.
